ckanext/validation/tests/helpers.py

```diff
@@ -42,74 +46,136 @@
-                'column-number': 3,
-                'message': 'Header in column 3 is blank',
-                'row': None,
-                'row-number': None
-            },
-            {
-                'code': 'duplicate-header',
-                'column-number': 4,
-                'message': 'Header in column 4 is duplicated to ...',
-                'row': None,
-                'row-number': None
-            },
-        ],
-        'headers': ['name', 'ward', 'party', 'other'],
-        'row-count': 79,
-        'source': 'http://example.com/invalid.csv',
-        'time': 0.007,
-        'valid': False,
-    }],
-    'time': 0.009,
-    'valid': False,
-    'warnings': [],
+    "valid": False,
+    "stats": {"tasks": 1, "errors": 1, "warnings": 0, "seconds": 0.004},
+    "warnings": [],
+    "errors": [],
+    "tasks": [{
+        "name": "4e-81c0-454e-9255-01bf50b096bb",
+        "type": "table",
+        "valid": False,
+        "place": "http://example.com/invalid.csv",
+        "labels": ["a", "b", "c", "d"],
+        "stats": {
+            "errors": 1, "warnings": 0, "seconds": 0.004,
+            "md5": "c451d87466e423fd10693865caeaa730",
+            "sha256": "b80b3082c86083ab623622bf299a17c793037fd5d3bbf0cdbca0c0438417de5a",
+            "bytes": 14,
+            "fields": 4,
+            "rows": 1
+        },
+        "warnings": [],
+        "errors": [{
+            "type": "missing-cell",
+            "title": "Missing Cell",
+            "description": "This row has less values compared to the header row (the first row in the data source). A key concept is that all the rows in tabular data must have the same number of columns.",
+            "message": "Row at position \"2\" has a missing cell in field \"d\" at position \"4\"",
+            "tags": ["#table", "#row", "#cell"],
+            "note": "",
+            "cells": ["1", "2", "3"],
+            "rowNumber": 2,
+            "cell": "",
+            "fieldName": "d",
+            "fieldNumber": 4
+        }]
+    }]
 }
 
+# GOODTABLES_ERROR_REPORT = {
+#     'error-count': 0,
+#     'table-count': 0,
+#     'warnings': ['Some warning'],
+# }
 ERROR_REPORT = {
-    'error-count': 0,
-    'table-count': 0,
-    'warnings': ['Some warning'],
+    "valid": False,
+    "stats": {"tasks": 1, "errors": 1, "warnings": 0, "seconds": 0.003},
+    "warnings": [],
+    "errors": [],
+    "tasks": [
+        {
+            "name": "file", "type": "table", "valid": False, "place": "http://example.com/file.csv", "labels": [],
+            "stats": {"errors": 1, "warnings": 0, "seconds": 0.003, "md5": "d41d8cd98f00b204e9800998ecf8427e",
+                      "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"},
+            "warnings": [],
+            "errors": [{
+                "type": "source-error",
+                "title": "Source Error",
+                "description": "Data reading error because of not supported or inconsistent contents.",
```

Comment:
Probably should say "unsupported" rather than "not supported".

Reply:
This is exactly from the frictionless source base, can't change it. just here to 'mimic' real errors in mocks.